src/components/pages/career-details/career-details-structured-data.tsx

```diff
@@ -1,12 +1,17 @@
 import React from 'react';
 import { Helmet } from 'react-helmet';
 import { SyPersonioJob } from '../../../types';
+import { JobPosting, WithContext } from 'schema-dts';
 
 interface CareerDetailsStructuredDataProps {
   position: SyPersonioJob;
 }
 
-const convertScheduleToEmploymentType = (schedule: string): string => {
+/**
+ * Convert the kebab-case, lowercase Personio values like `full-time`
+ * to the expected snake_case, uppercase format `FULL_TIME`.
+ */
+const normalizeJobScheduleFormat = (schedule: string): string => {
```

Comment:
the shitty personio api doesn't event liste all possible values. I hope this function does the mapping correctly.  I renamed it to highlight the purpose a little bit better.

Reply:
Google expects one of these:
- FULL_TIME
- PART_TIME
- CONTRACTOR
- TEMPORARY
- INTERN
- VOLUNTEER
- PER_DIEM
- OTHER

I guess this method would only work for the first two

Reply:
Well it would work for the other too, because no underscore is still okay for the algorithm. I just wanted to complain about Personio here 😄Resolved ✅